### PR TITLE
Added confirmation to git gud load next

### LINKED ISSUE #183: Confirm `git gud load next` if level is not complete
We can either do this via `git gud load next --force` (with the force option anywhere)

or we can do `confirm [y/n]`

Both should have relevant messages such as `Level not complete`

### PR DESCRIPTION
**Pull request by sahansk2**
Fixes #183

### CODE REVIEW

File: gitgud/__main__.py [hunk 1]
```diff
@@ -328,13 +330,21 @@ def handle_load(self, args):
...
         if args.skill_name:
             if args.skill_name.lower() in {"next", "prev", "previous"}:
                 query = args.skill_name.lower()
                 level = self.file_operator.get_level()
                                 
                 if query == "next":
-                    level_to_load = level.next_level
+                    current_level_complete = level._test(self.file_operator)
+                    if not args.force and not current_level_complete:
+                        print("It doesn't look like you've completed this level yet!")
```
Comment (benthayer) [inline gitgud/__main__.py:342]: Can you put this output into `skills/user_messages.py`

**Comment by benthayer** [comment]
The current output:
```
It doesn't look like you've completed this level yet!
Are you sure you want to go to the next level? If so, run `git gud load next` again with --force.
```
 seems a bit verbose, do you think you could come up with something shorter?

If you want to discuss, feel free to message me

**Comment by sahansk2** [comment]
Wait a second. I think we should wait to merge this until we have a framework to keep track of what levels users have completed as they go through git gud. Otherwise, if they're just going through the levels that they've completed in order again (maybe to look at the various trees or whatever, who knows), they'll be reminded that they haven't completed the level yet, which is not necessarily true. What do you think? @benthayer

**Comment by benthayer** [comment]
I think that's a good idea. Maybe this is a good time to start keeping track of which levels have been completed. 

I was holding off because I knew that the level names would change, but maybe it'd be worth giving each level a uuid so it becomes robust to name changes. Or we just wait.. What do you think?

**Comment by benthayer** [comment]
@sahansk2 see #32 for tracking completed levels. It is currently our oldest issue :+1:

**Comment by benthayer** [comment]
We are waiting on #241

File: gitgud/__main__.py [hunk 2]
```diff
@@ -406,6 +420,10 @@ def handle_load(self, args):
                 level = get_operator().get_level()
 
                 if args.skill_name == "next":
+                    current_level_complete = level.get_progress() == "complete"
```
Comment (benthayer) [inline gitgud/__main__.py:423]: Merge master and use `level.has_ever_been_completed`

File: gitgud/__main__.py [hunk 3]
```diff
@@ -175,9 +177,21 @@ def __init__(self):
...
-        levels_parser.add_argument('skill_name', metavar='skill', nargs='?')
-        levels_parser.add_argument('--all', dest='opt_all', action='store_true', help="Prints all available skills with levels.")  # noqa: E501
-        levels_parser.add_argument('--short', dest='opt_short', action='store_true', help="Prints with the short name of skills/levels usable with `git gud load`.")  # noqa: E501
+
+        levels_parser.add_argument(
+                'skill_name',
+                metavar='skill',
+                nargs='?')
+        levels_parser.add_argument(
+                '--all',
```
Comment (benthayer) [inline gitgud/__main__.py:186]: When merging master, you're going to need to add the `-a` option here

Comment (benthayer) [reply gitgud/__main__.py:187]: Done

**Pull request merged**